core/iwasm/interpreter/wasm_loader.c

```diff
@@ -6098,6 +6127,9 @@ wasm_loader_pop_frame_offset(WASMLoaderContext *ctx, uint8 type,
             ctx->dynamic_offset -= 2;
     }
     emit_operand(ctx, *(ctx->frame_offset));
+
+    (void)error_buf;
+    (void)error_buf_size;
```

Comment:
Not sure whether we need to apply similar changes to `wasm_mini_loader.c`?

Reply:
just some *unused variable* warning. I tend to modify `wasm_loader.c` only.